src/libstore/build.cc

```diff
@@ -2380,12 +2380,44 @@ void DerivationGoal::runChild()
 
             sandboxProfile += "(allow file-read* file-write-data (literal \"/dev/null\"))\n";
 
+            sandboxProfile += "(allow ipc-posix-shm*)\n";
+
+            sandboxProfile += "(allow mach-lookup\n"
+                "\t(global-name \"com.apple.SecurityServer\")\n"
+                ")\n";
+
+            sandboxProfile += "(allow file-read*\n"
+                "\t(literal \"/dev/dtracehelper\")\n"
+                "\t(literal \"/dev/tty\")\n"
+                "\t(literal \"/System/Library/CoreServices/SystemVersion.plist\")\n"
```

Comment:
That file is read by clang or ld whenever they do anything, so it's needed for every derivation. Should I put it on stdenv instead of in here?